src/sentry/api/event_search.py

```diff
@@ -316,6 +316,7 @@ class SearchVisitor(NodeVisitor):
         "p99",
         "failure_rate",
         "user_misery",
+        "user_misery_prototype",
```

Comment:
Initially named this `user_misery.prototype` but it started getting a bit messy because we use an aggregate alias that strips out special characters like `.` in the frontend and mapping it back to the `user_misery.prototype` was getting a bit annoying.

Reply:
This is temporary, and will very soon be just called `user_misery`